pramen-py/tests/test_metastore/test_metastore.py

```diff
@@ -676,3 +679,31 @@ def test_metastore_reader_get_latest_uppercase(
         ignore_row_order=True,
         ignore_column_order=True,
     )
+
+
+def test_metastore_reader_from_config(
+    spark,
+    repo_root,
+) -> None:
+    file_path = PurePath(
+        repo_root / "tests/resources/test_metastore.conf"
+    ).as_posix()
+    hocon_config = ConfigFactory.parse_file(file_path)
+    metastore = MetastoreReader(spark).from_config(hocon_config)
+
+    assert len(metastore.config) == 6
```

Comment:
Missed this one :)
```suggestion
    assert len(metastore.tables) == 6
```

Reply:
I remembering it. It will in next commit :)